README.md

```diff
@@ -60,7 +60,7 @@ It is setup with two primary machines: web and db. Together these two virtual ma
 4. Create a Magento 2 build available at m2.demo:
 
     ```bash
-    vagrant ssh web -- /server/vagrant/bin/m2setup.sh --sampledata --hostname=m2.demo
+    vagrant ssh web -- /server/vagrant/guest/bin/m2setup.sh --sampledata --hostname=m2.demo
```

Comment:
Great catch! However… the location you updated to is actually in the provisioner pool, and m2setup.sh should be available in you env PATH in `/usr/local/bin` so if you can update your PR to reflect this, that would be great. Should look something like this: `vagrant ssh web -- m2setup.sh --sampledata --hostname=m2.demo`?